### PR TITLE
Feature/adjust padding sizes

### PR DESCRIPTION
**Pull request by AustinGreer**
## Description

The padding sizes accross the pages in the Admin tools, judge page, saved pages, account page and support page were not consistent. I applied a consistent width and margin sizes across all of these components.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have removed unnecessary comments/console logs from my code
- [ ] I have made corresponding changes to the documentation if necessary (optional)
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [ ] No duplicate code left within changed files
- [ ] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by bukit3point0 — APPROVED** [review]
Looks good! All new lines look consistently modified and the formatting has been kept as well. Thanks!

**Review by kmcervan — APPROVED** [review]
This looks good! Nothing new to add.

File: src/components/pages/JudgeTable/_JudgeTableStyles.less [hunk 1]
```diff
@@ -4,26 +4,26 @@
   display: flex;
   flex-direction: column;
   justify-content: center;
-  align-items: center;
-  align-content: center;
-  width: 80%;
-  margin: 2rem 0 0 10rem;
+  // align-items: center;
+  // align-content: center;
```
Comment (elCarlosSantiago) [inline src/components/pages/JudgeTable/_JudgeTableStyles.less:8]: Some comments may be worth removing.

File: src/components/pages/SupportPage/_SupportPageStyles.less [hunk 2]
```diff
@@ -6,11 +6,10 @@
   width: 80vw;
   display: flex;
   flex-direction: column;
-  align-items: flex-start;
+  // align-items: flex-start;
```
Comment (elCarlosSantiago) [inline src/components/pages/SupportPage/_SupportPageStyles.less:9]: Here as well, please

**Review by salvare736 — APPROVED** [review]
Everything looks clear!

**Review by elCarlosSantiago — APPROVED** [review]
Great, clean work Austin. Only a couple of outstanding comments can be removed.

**Review by JamesWhite1911 — APPROVED** [review]
The changes look good!

**Comment by bukit3point0** [comment]
I see the removed unnecessary comments so I'm merging this.

**Pull request merged**